platform-includes/configuration/auto-session-tracking/javascript.mdx

```diff
@@ -1,32 +1,43 @@
-{/* TODO(v9): Update this for the new behaviour (sessions are persisted in sessionStorage or memory) */}
+<PlatformCategorySection supported={['browser']}>
 
+By default, the JavaScript Browser SDKs are sending sessions.
 We create a session for every page load. For single-page applications, we will create a new session for every navigation change (History API).
 
-We mark the session as:
-
-- crashed if an _unhandled error_ or _unhandled promise rejection_ bubbled up to the global handler.
-- errored if the SDK captures an event that contains an exception (this includes manually captured errors).
-
-To receive data on user adoption, such as users crash free rate percentage, and the number of users that have adopted a specific release, set the user on the [`initialScope`](/platforms/javascript/configuration/options/#initial-scope) when initializing the SDK.
-
-By default, the JavaScript SDKs are sending sessions. To disable sending sessions, set the `autoSessionTracking` flag to `false`:
+To disable sending sessions, disable the `BrowserSession` integration:
 
 ```javascript
 Sentry.init({
-  autoSessionTracking: false, // default: true
+  defaultIntegrations: (integrations) => {
+    return integrations.filter(
+      (integration) => integration.name !== "BrowserSession"
+    );
+  },
```

Comment:
@lforst I've had a customer try this but it didn't seems to be correct?

They are using [@sentry/nextjs@9.1.0](https://discord.com/channels/621778831602221064/1341815571221577728/1341829203678466058) and have tried this but indicated that it didn't work (as the types suggested to him: `defaultIntegrations?: false | Integration[];`). 

Is this still correct?

Reply:
Whoops I am an idiot. Should be `integrations: ...`